integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -201,6 +201,22 @@ protected static void wipeAllOpenSearchIndices(RestClient client) throws IOExcep
     }
   }
 
+  protected static void configureClient(RestClientBuilder builder, Settings settings)
```

Comment:
Is this used anywhere?

Reply:
https://github.com/Bit-Quill/opensearch-project-sql/blob/58c5c1f56901a731e340b826f08dd88ea4e01a1f/integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java#L95-L101